FRW/Modules/Login/ViewModel/PrivateKeyLoginViewModel.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  PrivateKeyLoginViewModel.swift
+//  FRW
+//
+//  Created by cat on 2024/8/19.
+//
+
+import CryptoKit
+import Flow
+import FlowWalletKit
+import Foundation
+import UIKit
+import WalletCore
+
+// MARK: - PrivateKeyLoginViewModel
+
+class PrivateKeyLoginViewModel: ObservableObject {
```

Comment:
If a class is not explicitly designed to be inheritable, it's better to declare `final`, that slightly improves method invocations